public/content/lessons/2019/windmills/index.mdx

```diff
@@ -1,9 +1,232 @@
+
+<Figure
+    image="usa_imo.png"
+/>
+
+The contest is essentially a test, split over two days, with three questions given over four and a half hours each day. The questions are all proofs, meaning you don’t simply find some numerical answer, you have to discover and articulate a rigorous line of reasoning to answer each difficult question, and they are scored on a scale from 0 to 7.
+
+<Figure
+    image="imo_q.png"
+/>
+
+Of interest to us today is the one from 2011, with 563 total participants representing 101 countries. I know what you’re thinking, and the answer is yes. Those do all happen to be prime numbers. But that’s not why this test was interesting, out of these prime problem solvers, only one, Lisa Saurmann from Germany, got a perfect score. The only thing standing between the next two runners up and a perfect score that year was problem #2. This problem is beautiful, and despite evading many of the world’s best mathematicians of their age, the solution is something which anyone watching this video can understand.
+
+## The problem
+
+So let’s begin by reading it carefully.
+
+> Let S be a finite set of at least two points in the plane
+
+Mmkay, as you read a question it’s often helpful to start drawing out an example for yourself.
+
+<Figure
+    image="intro_img1.png"
+/>
+
+> Assume that no three points of S are collinear
+
+In other words, you never have three points that line up. So you can probably predict that the problem involves drawing lines here in a way where three points on one line would mess things up.
+
+<Figure
+    image="intro_img2.png"
+/>
+
+> A windmill is a process that starts with a line l going through a single point P in S. The line rotates clockwise about the pivot P until the first time that the line meets some other point belonging to S.
+
+Again, while reading it’s helpful to draw out an example, so we’ve got this line pivoting around some point until it hits another.
+
+<Figure
+    image="intro_img3.png"
+    video="intro_img3.mp4"
+    show="video"
+/>
+
+> This point, Q, takes over as the new pivot, and the line now rotates clockwise about Q, until it next meets a point of S. This process continues indefinitely.
+
+<Figure
+    image="intro_img4.png"
+    video="intro_img4.mp4"
+    show="video"
+/>
+
+Alright, that’s kind of fun. We keep rotating and changing the pivot. You can see why they call it a windmill process, and you can also see why they specified that no three points lie on one line. You wouldn’t want to run into the ambiguity where you don’t know what pivot to switch to. Okay, so what’s the question?
+
+> Show that we can choose a point P in S and a line l going through P such that the resulting windmill uses each point of S as a pivot infinitely many times.
+
+<Figure
+    image="intro_img5.png"
+    video="intro_img5.mp4"
```

Comment:
I know I said ~10 seconds for clips, but this one might benefit from going on long enough for someone to be convinced that all the points will be hit infinitely many times.

Reply:
One possibility would be to speed [this clip](https://www.dropbox.com/home/3Blue1Brown/videos/2019/windmill/1440p60?preview=WindmillExample15Points.mp4) by 2x, or even 3x: